script/collect.ts

```diff
@@ -1,33 +1,43 @@
-#!/usr/bin/env node
-
-require('dotenv-safe').load()
```

Comment:
Can we keep this in a conditional, similar to https://github.com/electron/releases/blob/bde3285dfd5b97f0625b496154306a477b1f2b03/script/collect.js#L4-L6? If not, Let's update https://github.com/electron/i18n/blob/master/contributing.md#environment-setup.

Reply:
Or we can switch to `dotenv` instead of `dotenv-safe`